roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -8,6 +8,7 @@ epel_enabled: false
 dns_late: false
 
 common_required_pkgs:
+  - python-httplib2
```

Comment:
Is this package really necessary? It is not needed on any other distro out there, including CentOS 7 (which should be quite close to OL7...)

Reply:
No, sorry, I must have missed deleting that; or it came back